practice_3.5/evaluation_file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+test_dir="/opt/se2001/$(basename $(pwd))"
+
+for dir in $(ls $test_dir/ | grep "test_case" | sort); do
+	read cmd <$test_dir/$dir/input
+	cmd="$cmd" bash script.sh
+	cmd="$cmd" bash $test_dir/$dir/output
```

Comment:
@prassr  We should not have script to be executed in output

Reply:
This is the same code copied from replit, sir.
I will change it if necessary.